### PR TITLE
[SDK-core] Add StreamId to FlowUpdatedEvent when querying from Subgraph

### PR DESCRIPTION
**Pull request by kasparkallas**
Why?
Useful for UI-s.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3468,19 +3468,6 @@
     ethereumjs-util "7.1.5"
     stack-trace "0.0.10"
 
-"@superfluid-finance/ethereum-contracts@1.4.3":
```
Comment (0xdavinchee) [inline yarn.lock:3471]: what happened here? @kasparkallas

File: packages/sdk-core/src/events.ts [hunk 2]
```diff
@@ -77,6 +77,7 @@ export interface FlowUpdatedEvent extends EventBase {
     flowRate: string;
     flowOperator: string;
     deposit: string;
+    streamId: string; // Subgraph ID.
```
Comment (0xdavinchee) [inline packages/sdk-core/src/events.ts]: I would personally be a bit more explicit here and say something like `Subgraph Superfluid Stream ID`.
When I see Subgraph ID, I initially think of this (might just be me though): `QmVQcG7NusjRarbmL4717WoaaAx8mSynTrek73NmznYYp3` the IPFS hash of the deployed subgraph.

File: yarn.lock (hunk 1 above)
Comment (kasparkallas) [reply yarn.lock:3471]: It weirdly appeared here with a `yarn install`: https://github.com/superfluid-finance/protocol-monorepo/pull/1255#discussion_r1087562403 and now it disappeared again. :joy:

File: packages/sdk-core/src/events.ts (hunk 2 above)
Comment (kasparkallas) [reply packages/sdk-core/src/events.ts]: Changed: 
image

Comment (0xdavinchee) [reply packages/sdk-core/src/events.ts]: even better!

File: yarn.lock (hunk 1 above)
Comment (0xdavinchee) [reply yarn.lock:3471]: packages r pain 🥲

**Review by 0xdavinchee — APPROVED** [review]
comments addressed and tests passing - LGTM

**Pull request merged**